bb/tasks.clj

```diff
@@ -0,0 +1,35 @@
+(ns tasks
+  (:require [babashka.deps :as deps]
+            [babashka.fs :as fs]
+            [babashka.process :as p]
+            [clojure.string :as str]))
+
+(defn compile-native
+  "Compile library to standalone jar and a native executable program."
+  []
+  (let [graalvm-home (or (System/getenv "GRAALVM_HOME")
+                         (throw (Exception. "Please set GRAALVM_HOME.")))
+        java-home (str (fs/path graalvm-home "bin"))
+        deps-clj-version (slurp "resources/DEPS_CLJ_VERSION")]
+    (println "Building deps " deps-clj-version)
+    (p/shell "lein" "deps.clj" "-Spath" "-Sdeps" "{:deps {borkdude/deps.clj {:mvn/version \"0.0.1\"}}}")
+    (p/shell "lein with-profiles +native-image do clean, uberjar")
+    (let [native-image (str (fs/path graalvm-home "bin"
+                                     (if (fs/windows?) "native-image.cmd" "native-image")))]
```

Comment:
Same here

Reply:
This appears to refer to exactly the same area as the previous review comment, perhaps a duplicate? :)